### PR TITLE
Gitops section addition

### LINKED ISSUE #4442: [UX] GitOps: Enhance page with section highlighting What is GitOps resource
### Current Behavior
There is a need for a small section on [gitops overview page](https://layer5.io/cloud-native-management/gitops) that highlights the [What is GitOps resource article](https://layer5.io/resources/cloud-native/what-is-gitops).

### Implementation
1. A small design is needed in Figma
2. Implementation of that design 

### Acceptance Tests
1.  A new section highlights the What is GitOps resource and links to that page.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Gautam-04**
**Description**
I have designed a new section for GitOps.
Screenshot (58)

https://github.com/layer5io/layer5/assets/124668418/1fd886f4-cbdb-4a6d-b985-57ecfa68f854

This PR fixes #4442

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 512be31cdb050a94927674e1d00ebdee843e928d at: https://64d8e73875bf5b1ffc6e4c13--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@Gautam-04 Let's discuss it on the websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: src/sections/gitops/index.js [hunk 1]
```diff
@@ -44,6 +45,38 @@ const GitOpsPage = () => {
...
+                primary
+                className="learn-more-btn"
+                title="Read More"
+                url="https://layer5.io/resources/cloud-native/what-is-gitops"
+              />
+            </div>
+          </Col>
+          <Col className="hero-image" lg={6} md={6} xs={12} sm={10}>
+            <div className="image-container">
+              <img src={Github_Dark} className="logo" />
```
Comment (Udit-takkar) [inline src/sections/gitops/index.js:74]: Github_Dark logo only works in dark mode. we need to show different logo in light mode

Comment (Gautam-04) [reply src/sections/gitops/index.js:74]: I will get it resolved ☺️

**Comment by l5io** [comment]
🚀 Preview for commit 47a8466715cc308bbfb04d35273aa426e6410bf5 at: https://64de34f326417b1552843fb0--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9bb5d63852cc85a08b273408575a25c99ae4337e at: https://64de43a90f330f26a234dab4--layer5.netlify.app

File: src/sections/gitops/index.js [hunk 2]
```diff
@@ -38,6 +40,38 @@ const GitOpsPage = () => {
...
+      <div className="content-container content-container-more-width">
+        <Row className="content-row">
+          <Col lg={6} md={6} className="desc-text-container">
+            <h1 className="heading-1">What is GitOps?</h1>
+            <SquarePoint>
+              GitOps uses Git for defining and tracking infrastructure and app
+              changes.
+            </SquarePoint>
+            <SquarePoint>
+              It automates syncing actual state with Git's desired state.
```
Comment (harshsingh32) [inline src/sections/gitops/index.js:52]: It’s an evolution of Infrastructure as Code (IaC) that leverages Git as the single source of truth, and control mechanism for creating, updating, and deleting system architecture.

These lines can be added.

**Review by harshsingh32 — APPROVED** [review]
some lines can be added.

File: src/sections/gitops/index.js (hunk 2 above)
Comment (Gautam-04) [reply src/sections/gitops/index.js:52]: Ohk i will add them

**Review by Chadha93 — APPROVED** [review]
Thank you gents, lets cover content update separately.

**Pull request merged**